Game/Config/MapSettingsOverrides/TestOverridesSpatialEventTracingGroup.ini

```diff
@@ -1,8 +1,5 @@
 [/Script/SpatialGDK.SpatialGDKSettings]
 bEventTracingEnabled=True
 
-[/Script/SpatialGDKEditor.SpatialGDKEditorSettings]
-SpatialOSCommandLineLaunchFlags="--event-tracing-enabled=true"
```

Comment:
:o Is this defo correct?

Reply:
yes, this is removed with Danny's new event tracing work, its odd merge bc there were two files turned into 1 in master, while one of those files had these two lines removed in 13.1